ext/templates/deb/postinst.erb

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
-<% if @pe -%>
-export PATH=/opt/puppet/bin:$PATH
-<% end -%>
```

Comment:
Is this ok to remove because there's a pe-puppet fork? If so, could you update the commit message?